httpbin/handlers.go

```diff
@@ -517,9 +517,8 @@ func (h *HTTPBin) Stream(w http.ResponseWriter, r *http.Request) {
 	f := w.(http.Flusher)
 	for i := 0; i < n; i++ {
 		resp.ID = i
-		line, _ := json.Marshal(resp)
+		line, _ := jsonMarshalNoEscape(resp)
 		w.Write(line)
-		w.Write([]byte("\n"))
```

Comment:
This was needed because `json.Marshal` doesn't add a newline while `.Encode` does. I confirmed that httpbin seems to return a newline at the end of responses